src/containers/Form/FormLayout.tsx

```diff
@@ -216,6 +222,38 @@ export const FormLayout: React.SFC<FormLayoutProps> = ({
     return null;
   }
 
+  const getMediaId = (payload: any) => {
```

Comment:
I think this is not something we should include in FormLayout.tsx. Maybe in parent or maybe we create another (generic) wrapper around FormLayout that deals with media. Let me know your thoughts.

Reply:
I have fixed this, please review.